spec/reactors/mempool/messages.md

```diff
@@ -0,0 +1,52 @@
+# Mempool Messages
+
+## P2P Messages
+
+There is currently only one message that Mempool broadcasts and receives over
+the p2p gossip network (via the reactor): `TxsMessage`
+
+```go
+// TxsMessage is a MempoolMessage containing a list of transactions.
+type TxsMessage struct {
+    Txs []types.Tx
+}
+```
+
+## RPC Messages
```

Comment:
This is more documentation than spec, I'd say.

We can check the related content there to merge.